main/src/ui/notifier_freedesktop.vala

```diff
@@ -120,6 +121,7 @@ public class Dino.Ui.FreeDesktopNotifier : NotificationProvider, Object {
         hash_table["sound-name"] = new Variant.string("phone-incoming-call");
         hash_table["urgency"] = new Variant.byte(2);
         hash_table["desktop-entry"] = new Variant.string(Dino.Application.get_default().get_application_id());
+        hash_table["category"] = new Variant.string("im");
```

Comment:
Isn't there anything more suitable for calls?

Reply:
Not really: https://specifications.freedesktop.org/notification-spec/latest/ar01s06.html